GOVERNANCE.md

```diff
@@ -72,6 +72,19 @@ depend on and trust to make decisions in the best interest of the project.
 
 Maintainers have admin access to the GitHub organization.
 
+### Chairs
+
+OpenEBS Chairs are ex-maintainers that have stepped down from active contributions, 
```

Comment:
ex-maintainers and decision-makers

Reply:
maintainers are responsible for decisions. Probably we don't need to call it out?

Reply:
The wording says "ex-maintainers", hence the comment.